dali/python/nvidia/dali/plugin/base_iterator.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali import types
+import math
+import logging
+import numpy as np
+
+class _DaliBaseIterator(object):
+    def __init__(self,
+                 pipelines,
+                 size=-1,
+                 reader_name=None,
+                 auto_reset=False,
+                 fill_last_batch=True,
```

Comment:
A bit nitpicky, but I need to look the arguments in derived class to see what is it about. Maybe copy the "base" comment here?

Reply:
Done